src/main/java/com/snowflake/kafka/connector/internal/SnowflakeSinkServiceV1.java

```diff
@@ -815,18 +797,13 @@ private long getOffset() {
 
       // This api should throw exception if backoff failed.
       // fileNamesCopy after this call is emptied (clears the input list)
-      if (ingestionDeliveryGuarantee
-          == SnowflakeSinkConnectorConfig.IngestionDeliveryGuarantee.EXACTLY_ONCE) {
-        ingestionService.ingestFilesWithClientInfo(fileNamesCopy, this.clientSequencer.get());
-        String offsetToken = ingestionService.getClientStatus().getOffsetToken();
-        // Update server side offset
-        if (offsetToken == null) {
-          this.offsetPersistedInSnowflake.set(-1);
-        } else {
-          this.offsetPersistedInSnowflake.set(Long.parseLong(offsetToken));
-        }
+      ingestionService.ingestFilesWithClientInfo(fileNamesCopy, this.clientSequencer.get());
+      String offsetToken = ingestionService.getClientStatus().getOffsetToken();
```

Comment:
we wont be calling this API anymore, we should remove offsetPersistedInSnowflake field too.

Reply:
just curious - why are we not calling this API anymore? and why is the offsetPersistedInSnowflake field not necessary? I'm not entirely sure what the importance of this field is/was